### ISSUE TITLE
No 404 Page

### ISSUE THREAD

**Issue by obvMath** [Externalizable/bongo.cat#417]
Hey Eric! Can you add a custom 404 page, I think it would improve the website. Hit me up if you need help, I know how :D

**Comment by Externalizable** [comment]
I agree, the default 404 page provided by GitHub Pages is a little off-putting.

Earlier today I deployed a minor change to the Cloudflare Worker handling incoming requests to bongo.cat so that it now redirects (and edge caches) any responses throwing 404s.

Obviously force-redirecting all 404s is not ideal and eventually, a simple 404 page with an automatic redirect to the home page is preferred.
